go-controller/pkg/ovn/egressgw.go

```diff
@@ -47,7 +47,9 @@ type ovnRoute struct {
 }
 
 type externalRouteInfo struct {
-	sync.RWMutex
+	sync.Mutex
+	deleted bool
+	podName ktypes.NamespacedName
```

Comment:
q: why are we adding this podName here? we have the key on main cache as ns/podName and podExternalRoutes has the pod.IPs as its key. Is it only for logging purposes?

Reply:
You're right, don't need it here, it was left-over from the cleanup PR.